osrf_pycommon/process_utils/get_loop_impl.py

```diff
@@ -24,30 +23,34 @@ def get_loop_impl(asyncio):
     if getattr(_thread_local, 'loop_has_been_setup', False):
         return asyncio.get_event_loop()
```

Comment:
```suggestion
        try:
            return asyncio.get_running_loop()
        except AttributeError:
            # Thrown when on Python < 3.7
            return asyncio.get_event_loop()
```

I believe this section of the code should also look like this. However, I noticed that at this point of the code the method returns a `RuntimeError` which shouldn't happen according to my understanding. It would seem there is a bug at setting a `new_event_loop` to `asyncio`. Let me know your thoughts @clalancette